rest-openehr/pom.xml

```diff
@@ -35,6 +35,8 @@
 
     <properties>
         <license-header.dir>../</license-header.dir>
+        <ipf-atna-spring-boot-starter.version>4.0.0</ipf-atna-spring-boot-starter.version>
```

Comment:
`ipf.version` should be sufficient; the version should be managed in the server pom, or maybe in the bom.